docs/_posts/2022-12-21-jekyll-sass-converter-3.0-released.markdown

```diff
@@ -0,0 +1,71 @@
+---
+title: Jekyll Sass Converter 3.0 Released
+date: 2022-12-21 17:52:15 +0530
+author: ashmaroli
+category: community
+---
+
+Jekyll Sass Converter 3.0 shipped recently, is available to those using Jekyll 4.3 and above and contains major changes.
```

Comment:
I appreciate how dense this sentence is, but I think it'll read better if we split it up and add a bit more context

```suggestion
Jekyll Sass Converter 3.0 shipped recently and is available to those using Jekyll 4.3 and above. This release contains major changes.
```